moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -301,7 +301,7 @@ class TrajectoryExecutionManager
   void loadControllerParams();
 
   // Name of this class for logging
-  const std::string name_ = "trajectory_execution_manager";
+  const std::string LOGNAME = "trajectory_execution_manager";
```

Comment:
An upper-case variable indicates a global or static one. Thus, this should be changed to `static` as well:
```suggestion
  static const std::string LOGNAME = "trajectory_execution_manager";
```

Reply:
sure